### PR TITLE
Loaded TextureAtlas via Gestalt

### PR DESCRIPTION
**Pull request by vampcat**
Most of the changed files are shifting/renaming, with a few actual changes.

- Addition of src/Assets/TextureMap package
- Modification of sol.atlas to make the texture names more sensible.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: main/src/org/destinationsol/assets/TextureMap/TextureMap.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * limitations under the License.
+ */
+package org.destinationsol.assets.TextureMap;
+
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import org.terasology.assets.Asset;
+import org.terasology.assets.AssetType;
+import org.terasology.assets.ResourceUrn;
+
+public class TextureMap extends Asset<TextureMapData> {
```
Comment (Cervator) [inline main/src/org/destinationsol/assets/TextureMap/TextureMap.java]: Maybe we can get started right and try to introduce javadoc on all new classes like this, even if they're basic :D

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Tested out with everything latest and merged! :-)

Good to see lots of cleanup. I'm noting some minor issues (like the beacon sfx plays continuously when "docked"? Huh) but I suspect partly that's from never having had the opportunity to notice before + we'll be hitting a lot of stuff in the near future so not going to bother with issues just yet.

**Pull request merged**